build.gradle

```diff
@@ -77,6 +91,14 @@ static String buildNumberToIntellijVersion(String buildNumber) {
   return '20' + buildNumber.substring(0, 2) + '.' + buildNumber.substring(2, 3)
 }
 
+static Integer buildNumberToNumericValue(String buildNumber) {
```

Comment:
Hmmm I'm skeptical about this conversion... and esp. about comparing EAP versions in such a way!

Consider comparing `222.137.42` with `222.99.999`.
Although it's the **former** that's actually higher (coz 137 > 99), this approach will consider the **latter** higher (coz 22213742 < 22299999) :/

Consider https://stackoverflow.com/a/36400405/2750812 (it'll serve at least as long as we're on Gradle 7.x, later to be re-implemented/replaced with sth else)

Reply:
a okey in fact, I did not include this case :/